src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -82,6 +82,13 @@ private function configureRoutes(RoutingConfigurator $routes): void
         } else {
             $routes->import($configDir.'/{routes}.php');
         }
+
+        if (
+            false !== ($fileName = (new \ReflectionClass(static::class))->getFileName()) &&
+            is_file($fileName)
+        ) {
+            $routes->import($fileName, 'annotation');
```

Comment:
attribute instead of annotation, since that works now?

Reply:
`attribute` would require to bump symfony/routing isn'it?

Reply:
Yes, changing to `attribute` breaks the "lowest" build. I'm not sure if this is worth a bump.